<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>armToolUI</class>
 <widget class="QWidget" name="armToolUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0,0);
background-image:url(://Service_Tools/Service_Tools/arm_background.png);
border: 0px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>65</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://Service_Tools/Service_Tools/tilting_tool.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="temp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
     </property>
     <property name="text">
      <string notr="true">---(Â°C)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>10</y>
      <width>616</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">Service Panel/Tools/Arm motor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="in3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>235</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="in4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>215</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="in2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>255</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="in1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>275</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="vpower">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>185</y>
      <width>96</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---(V)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vlogic">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>295</y>
      <width>96</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---(V)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>425</y>
      <width>741</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,30);
background-image:url(://transparent.png);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="vpower_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>1</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);
background-color:rgb(0,0,0,0);</string>
     </property>
     <property name="text">
      <string notr="true">ERRORS:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="errClass">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>2</y>
       <width>626</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);
background-color:rgb(0,0,0,0);</string>
     </property>
     <property name="text">
      <string notr="true">ssss</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="errSubclass">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>19</y>
       <width>626</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);
background-color:rgb(0,0,0,0);</string>
     </property>
     <property name="text">
      <string notr="true">ssss</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="errCode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>36</y>
       <width>626</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);
background-color:rgb(0,0,0,0);</string>
     </property>
     <property name="text">
      <string notr="true">ssss</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
